common-content/en/module/html-css/what-is-html/index.md

```diff
@@ -133,7 +133,13 @@ HTML tags are powerful because they are meaningful. They are meaningful because
 
 HTML tags are powerful because they are meaningful. They are meaningful because they are semantic. Semantic HTML is powerful HTML.
 
-The better structured your document, the more meaning you can pack into it, the more powerful your code is, the better it will work in more contexts, and more things will be able to interface with it. The more meaning you put in, the more meaning other readers can get out.
+The better structured your document, the more meaning you can pack into it, the more powerful your code is, the better it will work in more contexts, and more things will be able to interface with it.
+
+One place semantic HTML is useful is that if you use a `<footer>` tag instead of a `<div>` tag, a screen reader can understand that it is a header, and treat it specially.
```

Comment:
```suggestion
One place semantic HTML is useful is that if you use a `<footer>` tag instead of a `<div>` tag, a screen reader can understand that it is a footer, and treat it specially.
```